scipy/stats/_continuous_distns.py

```diff
@@ -7843,17 +7843,38 @@ def _shape_info(self):
         return [ic, i_s]
 
     def _pdf(self, x, c, s):
-        # powerlognorm.pdf(x, c, s) = c / (x*s) * phi(log(x)/s) *
-        #                                         (Phi(-log(x)/s))**(c-1),
-        return (c/(x*s) * _norm_pdf(np.log(x)/s) *
-                pow(_norm_cdf(-np.log(x)/s), c*1.0-1.0))
+
+        def direct_formula(x, c, s):
+            return (c/(x*s) * _norm_pdf(np.log(x)/s) *
+                    pow(_norm_cdf(-np.log(x)/s), c - 1.))
+
+        def logpdf_formula(x, c, s):
+            return np.exp(self._logpdf(x, c, s))
+
+        return _lazywhere((c <= 1) & (s <= 1), (x, c, s, ),
+                          logpdf_formula,
+                          f2=direct_formula)
```

Comment:
Is there a big advantage to the direct formula here, or can this all just be exponentiating the `_logpdf`?

Reply:
Without the `_logpdf`, the range of the PDF is definitely limited but slightly more accurate. I think though that an increased range is more important, so removed the direct formula for now. A follow up could figure out a good cutoff between both.

![image](https://user-images.githubusercontent.com/40656107/226098938-cd641646-ddb3-43df-9736-cab314aa3031.png)